schedule/ha/bv/ha_supportserver.yaml

```diff
@@ -22,7 +22,6 @@ vars:
   DESKTOP: textmode
   QEMU_DISABLE_SNAPSHOTS: '1'
   SUPPORT_SERVER: '1'
-  SUPPORT_SERVER_ROLES: dhcp,dns,ntp,ssh,iscsi
```

Comment:
This variable needs to be set in the job group definition because for the migration verify tests, we don't need `iscsi` as a role.

Reply:
Ok, please just modify the SAP one at the same time :)

Reply:
For SAP, AFAIK, your SUPPORT_SERVER_ROLES variable is always set to dhcp,dns,ntp,ssh,iscsi and you use another file. So I don't think we need to change something there.
However, changes are needed for ha_preflightcheck_supportserver.

Reply:
Yes it was for preflight, I'm too tired :D